src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  //The selected button is disabled to let the user know which data he is viewing (jobs or training).
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  // If a button is disabled it means it's data is already fetched. So, on the click of the other button this function detects the disabled button and fetches the opposite data.
+  const fetchOpportunityData = () => {
+    if (isDisabled) {
+      setSearchCards([...t(`home.trainings`, { returnObjects: true })]);
+      setCardsArr([...t(`home.trainings`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    } else {
+      setSearchCards([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setCardsArr([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
```

Comment:
line 15 and 19 are repeated, you can move it below line 20 outside teh conditions

Reply:
done

Reply:
done